packages/echo-base/src/module/fetch.ts

```diff
@@ -4,11 +4,11 @@
  * @param token The Token for authentication.
  * @returns A promise leading to the raw text content.
  */
-export async function fetchDependency(url: string, token: string): Promise<string> {
-    const m = await fetch(url, {
+export async function defaultFetchDependency(url: string, token?: string): Promise<string> {
```

Comment:
Dette er kun en fetch, og har ikke noe specifikt for depedency. Rename til noe sånt som fetchUrl eller fetchTextFromUrl

Eventuelt, hvorfor ikke bruke fetch som vi har allerede?

Reply:
Tror denne kan slettes egentlig, skal sjekke opp ;)

Reply:
kunne slettes ;)